apps/dashboard/src/components/workflow-editor/steps/conditions/edit-step-conditions-skeleton.tsx

```diff
@@ -0,0 +1,39 @@
+import { RiQuestionLine } from 'react-icons/ri';
+import { Link } from 'react-router-dom';
+import { RiInputField } from 'react-icons/ri';
+
+import { Button } from '@/components/primitives/button';
+import { Panel, PanelContent, PanelHeader } from '@/components/primitives/panel';
+import { Skeleton } from '@/components/primitives/skeleton';
+
+export const EditStepConditionsFormSkeleton = () => {
+  return (
+    <form className="flex h-full flex-col">
```

Comment:
A form isn't necessary in the skeleton, we can remove it.

Reply:
it's not